modules/creating-a-custom-training-image.adoc

```diff
@@ -0,0 +1,118 @@
+endif::[]
+
+* You can access the base training image.
+* You have Podman installed in your local environment, and you can access a container registry.
++
+For more information about Podman and container registries, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/building_running_and_managing_containers/index[Building, running, and managing containers].
+
+
+.Procedure
+
+. In a terminal window, create a directory for your work, and change to that directory. 
++
+In the example commands in this section, `my_training_image` is the name of the working directory _and_ the name of the image you create.
++
+.Create a working directory
+[source,bash]
+----
+export IMG=my_training_image
+mkdir $IMG
+cd $IMG
+----
+
+. Create a file named `Dockerfile` with the following content:
+
+.. Use the `FROM` instruction to specify the base training image location.
+
+* To create a CUDA-compatible Ray cluster image, specify the CUDA-compatible Ray base image location:
++
+[source,bash]
+----
+FROM quay.io/modh/ray:2.35.0-py39-cu121
+----
+
+* To create a ROCm-compatible Ray cluster image, specify the Developer Preview ROCm-compatible Ray base image location:
++
+[source,bash]
+----
+FROM quay.io/modh/ray:2.35.0-py39-rocm61
+----
++
+[IMPORTANT]
+====
+The ROCm-compatible Ray base cluster image is Developer Preview software only. 
+Developer Preview software is not supported by {org-name} in any way and is not functionally complete or production-ready. 
+Do not use Developer Preview software for production or business-critical workloads. 
+Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a {org-name} product offering. 
+Customers can use this software to test functionality and provide feedback during the development process. 
+This software might not have any documentation, is subject to change or removal at any time, and has received limited testing. 
+{org-name} might provide ways to submit feedback on Developer Preview software without an associated SLA.
+
+For more information about the support scope of {org-name} Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
+
+
+.. Use the `RUN` instruction to install additional packages.
+You can also add comments to the Dockerfile by prefixing the comment lines with a number sign (`#`).
++
+The following example shows how to install a specific version of the Python PyTorch package:
++
+.Install additional packages
+[source,bash]
+----
+# Install PyTorch
+RUN python3 -m pip install torch==2.4.0
+----
+
+
+. Build the image file. 
+Use the `-t` option to create an image tag that specifies the image name and version, to make it easier to reference and manage the image: 
```

Comment:
Use the `-t` option to...
 >
Use the `-t` option with the `podman build` command to...